src/main.js

```diff
@@ -9,8 +10,9 @@ const filtersContainer = tripMainContainer.querySelector('.trip-controls__filter
 const tripEventsContainer = document.querySelector('.trip-events');
 
 const tripPresenter = new TripPresenter();
+const eventModel = new EventModel();
```

Comment:
Следует подправить нейминг это модель ивентов, а не одного ивента
```suggestion
const eventsModel = new EventsModel();
```